Jupyter Notebook/Computer Vision API Example.ipynb

```diff
@@ -51,8 +51,8 @@
    "outputs": [],
    "source": [
     "# Variables\n",
-    "\n",
-    "_url = 'https://westcentralus.api.cognitive.microsoft.com/vision/v1/analyses'\n",
+    "_region = 'westus' #Here you enter the region of your subscription\n",
```

Comment:
Is there a good reason for this not being `westcentralus`?

Reply:
Um in general it seems to me that the default value for the region (or at least the most used) is westus. But I can change it back if you want to.

Reply:
Please do.  This was an intentional [change](https://github.com/Microsoft/Cognitive-Vision-Windows/commit/1da8394bff5441ec07cfcb32cc8ed4fe75efef29).

Reply:
Done!